index.js

```diff
@@ -9,6 +9,8 @@ const readdirSync = require('fs').readdirSync
 const resolve = require('path').resolve
 const join = require('path').join
 const basename = require('path').basename
+const dirname = require('path').dirname
+const extname = require('path').extname
```

Comment:
why not refactoring using destructuring syntax? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Yes. I considered it, but I was just trying to preserve the same structure. So, I'll modify it.

> why not refactoring using destructuring syntax? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment